modules/data.land/R/extract_veg.R

```diff
@@ -67,14 +67,17 @@ extract_veg <- function(new_site, start_date, end_date,
    if(source == "FIA"){
    format_name <- "fia" 
    code_col    <-  "spcd"
+ } 
+ if(source == "NEON_veg"){
+   format_name = "usda"
+   code_col = "species_USDA_symbol"
    }else{
       code_col    <- "species_USDA_symbol"
       format_name <- "usda"
       obs[obs$species_USDA_symbol != "2PLANT", ] #removes the rows with 2PLANT, this is a NEON specific code that means they could not identify the species 
    }
+>>>>>>> f5194f84b26a85bbfcb279cc4665d7560205efc7
```

Comment:
This is indicative of a merge conflict and needs to be resolved

Reply:
Yes I am going to revert this commit